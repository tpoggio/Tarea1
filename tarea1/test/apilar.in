esVaciaPila
apilar 15 5.2
esVaciaPila
esVaciaPila
Fin