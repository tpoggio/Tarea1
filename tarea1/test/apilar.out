1>La pila está vacia. 
2>Apilar. 
3>La pila no está vacia. 
4>La pila no está vacia. 
5>Fin.
